frontend/.env.dev

```diff
@@ -1,5 +1,6 @@
 NODE_ENV=development
-VUE_APP_BACKEND_URL=https://qa-api.notificasaude.com.br/api
+VUE_APP_BACKEND_URL=http://localhost:3000/api
 VUE_APP_KEYCLOAK_URL=https://qa-auth.notificasaude.com.br/auth
 VUE_APP_KEYCLOAK_REALM=notificasaude
 VUE_APP_KEYCLOAK_CLIENT_ID=notificasaude
+VUE_APP_MUNICIPIO_ID=ac3227a1-8a09-4b5f-93cd-d6ca43b637a2
```

Comment:
Por favor, converse com o @wildeee a respeito dessa variável. Recentemente ele o alterou

Reply:
Muito bem apontado senhor @ftathiago 
@carloshabarbosa, como fazemos o mesmo para o Keycloak nesse arquivo, gostaria de manter como padrão de desenvolvimento o front apontando para o ambiente de homologação.
Caso seja necessário pra você fazer testes locais, crie um .env.dev.local sobrescrevendo as variáveis desejadas, como consta na [documentação oficial do vue](https://cli.vuejs.org/guide/mode-and-env.html#environment-variables).

Reply:
Verdade, esqueci de remover esse cara do commit, eu estava alterando esse arquivo para rodar local. Obrigado pelas dicas